### PR TITLE
grammar and formatting

### PR DESCRIPTION
**Pull request by tjperry07**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,61 +132,66 @@ There are four configs within the **Node Properties** group.
...
 
 * **Grouping Type**:
-  * Key Based:The incoming source data is grouped based on the bucket column as the key.
-  * Range Buckets:The incoming source data is grouped based on the bucket column into different  groups mentioned in 'Number of Buckets' textbox.
-    * Number of Buckets :The number of groups based on which data will be divided.
-* **Bucket Columns**:
+  * Key Based: The incoming source data is grouped based on the bucket column as the key.
+  * Range Buckets: The incoming source data is grouped based on the bucket column into different  groups mentioned in 'Number of Buckets' textbox.
+  * Number of Buckets: The number of groups based on which data will be divided.
+* **Number of Groups**: [Missing]
```
Comment (tjperry07) [inline README.md]: @jgontarz @prj2001-udn There seems to be a definition missing here

**Review by prj2001-udn — APPROVED** [review]
The review points verified and corrected

File: README.md (hunk 1 above)
Comment (prj2001-udn) [reply README.md]: The missing information has been added

**Review by prj2001-udn — COMMENTED** [review]
Missing content added

**Pull request merged**